ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.template.html

```diff
@@ -28,14 +28,17 @@ <h2 class="spec-title">
                     </h2>
 
                     <div class="btn-group pull-right spec-actions" uib-dropdown>
-                        <button id="spec-actions" type="button" class="btn btn-link" ng-if="model.hasType()" uib-dropdown-toggle>
+                        <button id="spec-actions" type="button" class="btn btn-link" ng-if="model.hasType() && allowOpenInCatalog" uib-dropdown-toggle>
                             <span class="fa fa-bars"></span>
                         </button>
                         <ul class="dropdown-menu" uib-dropdown-menu role="menu" aria-labelledby="spec-actions" ng-if="model.hasType()">
```

Comment:
maybe we could add allowOpenInCatalog here too? If allowOpenInCatalog == false we have some useless (not displayed) nodes in the DOM